src/rp2_common/hardware_pwm/include/hardware/pwm.h

```diff
@@ -134,6 +134,21 @@ static inline void pwm_config_set_clkdiv_int(pwm_config *c, uint div) {
     c->div = div << PWM_CH0_DIV_INT_LSB;
 }
 
+/** \brief Set PWM clock divider in a PWM configuration using an 8:4 fractional value
+ *  \ingroup hardware_pwm
+ *
+ * \param c PWM configuration struct to modify
+ * \param integer 8 bit integer part of the clock divider. Must be greater than or equal to 1.
+ * \param fract 4 bit the fractional part of the clock divider
```

Comment:
Might be worth chucking in a `valid_params_if` to check that `fract` isn't too big?

Reply:
And I suspect "4 bit the fractional part..." ought to be "The 4-bit fractional part..."

Reply:
Ah good idea. I see this is included in the existing function, but I didn't think to copy it over
https://github.com/raspberrypi/pico-sdk/blob/d54104a1e5cff04979fe442cadd3018ac583d5d2/src/rp2_common/hardware_pwm/include/hardware/pwm.h#L379

Reply:
Done